act3.sh

```diff
@@ -1,13 +1,17 @@
 # 1. Unload the Millenium Falcon in ONE COMMAND!
 # Move the whole crew from the millenium_falcon directory into the death_star directory. HINT: * following a directory will grab all files/folders inside of a directory (directory/*)
+mv ~/sei/deliverables/Star_Wars/empire/death_star/millenium_falcon/* ~/sei/deliverables/Star_Wars/empire/death_star
```

Comment:
Great use of `*`. This also looks like a direct path. Try using relative paths moving forward! :)